.github/workflows/spread-manual.yaml

```diff
@@ -43,7 +43,13 @@ jobs:
         with:
           name: snap
           path: tests
-      - name: Run spread (large/flaky)
+
+      - name: Run remote-build tests
+        env:
+          LAUNCHPAD_TOKEN: "${{ secrets.LAUNCHPAD_TOKEN }}"
```

Comment:
Note that this secret won't be available for fork-based PRs ([ref](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflows-in-forked-repositories))

Reply:
(i wouldn't make a big deal out of it though because this also runs on a schedule)

Reply:
yeah that's fine. After the feature stabilizes I don't expect this to break often (well, not from changes on our side)
also that token is a little too powerful so the fact that it's not available to forks is a good thing

Reply:
these tests are also not triggered with every PR